src/Microsoft.IdentityModel.Tokens/Telemetry/TelemetryClient.cs

```diff
@@ -21,51 +21,55 @@ internal class TelemetryClient : ITelemetryClient
             AppContextSwitches.UpdateConfigAsBlocking.ToString()
         );
 
-        public void IncrementConfigurationRefreshRequestCounter(string metadataAddress, string operationStatus)
+        public void IncrementConfigurationRefreshRequestCounter(string metadataAddress, string operationStatus, string configurationSource)
```

Comment:
TelemetryClient is used in other projects, so updating these methods can cause the version mismatch issues. What about creating additional overload methods instead that pass default config source?

Reply:
Resolving in favor of https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/3254#discussion_r2121747992